client/pokemon/pokemon-icon.view.html

```diff
@@ -0,0 +1,4 @@
+<div class="icon-container" ng-init="pokemon.parseCardViewProps()" flex
+  ng-class="['spr-box', pokemon.spriteClass, {'pokemon-egg': pokemon.data.isEgg}]" >
+  <span ng-if="::pokemon.data.isEgg" class="spr-box spr-box-egg"></span>
```

Comment:
I'm not sure if we want the egg in that position, i.e. covering the Pokémon's face. Hence the uncertainty as to whether we should consider that a complete fix or not.

For comparison, we usually place the egg sprite [behind the Pokémon](http://i.imgur.com/g3SCADy.png), but we might not have enough room to keep it noticeable on a grid like this.